packages/compiler/test/test-parser-utils.ts

```diff
@@ -26,8 +23,8 @@ describe("compiler: parser utils", () => {
         }
       `);
       ok(node);
-      strictEqual(node.kind, SyntaxKind.NamespaceStatement);
-      strictEqual((node as NamespaceStatementNode).id.sv, "Foo");
+      strictEqual(node.kind, SyntaxKind.NamespaceStatement as const);
```

Comment:
does this make it that the type inference work after? This is awesome, so anyoing to have to the casting after

Reply:
Yes, I expected it to work because I thought I did it elsewhere. But turns out I was doing ok(node.kind === SyntaxKind.X, "Expected X"). This is better. :)